fix deployment_tools in docs (#3364)
openvinotoolkit/open_model_zoo: tools/accuracy_checker/configs/README.md

    ```sh

>     OMZ_ROOT/tools/model_tools/downloader.py --name densenet-121-tf --output_dir MODEL_DIR

Reviewer: ```suggestion
    omz_downloader --name densenet-121-tf --output_dir MODEL_DIR
```

    ```
3. Convert model in the OpenVINO IR format using Model Optimizer via [Model Converter](../../../tools/model_tools/README.md)
    ```sh
    omz_converter --name densenet-121-tf --download_dir MODEL_DIR
    ```
4. Run evaluation for model in FP32 precision using [Accuracy Checker](../README.md)

>     ```sh

Reviewer 2: Should we use omz_downloader and  omz_quantizer aliases instead of scripts everywhere on this page?

Author: yes, we should

    ```

> 3. Convert model in the Inference Engine IR format using Model Optimizer via [Model Converter](../../../tools/model_tools/README.md)

Reviewer: ```suggestion
3. Convert model in the OpenVINO IR format using Model Optimizer via [Model Converter](../../../tools/model_tools/README.md)
```

Author: accepted

    ```sh

>     OMZ_ROOT/tools/model_tools/converter.py --name densenet-121-tf --download_dir MODEL_DIR

Reviewer: ```suggestion
    omz_converter --name densenet-121-tf --download_dir MODEL_DIR
```

Author: accepted
